api/src/domain/license-status/searchBusinessAddressesForMatch.ts

```diff
@@ -0,0 +1,38 @@
+import { LicenseSearchAddress } from "@shared/license";
+import { BusinessIdToAddressesMapping } from "../types";
+
+export const searchBusinessAddressesForMatch = (
+  businessAddresses: BusinessIdToAddressesMapping[],
+  nameAndAddress: LicenseSearchAddress
+): string => {
+  const result: string[] = [];
+
+  const isAddressMatch = (
+    address: LicenseSearchAddress,
+    userInputtedAddress: LicenseSearchAddress
+  ): boolean => {
+    return (
+      userInputtedAddress.addressLine1 === address.addressLine1 &&
+      userInputtedAddress.addressLine2 === address.addressLine2 &&
```

Comment:
[pebble] Our existing logic does not match on addressLine2 at all (it is ignored completely) - we only match on addressLine1 and zipCode. We should keep this same behavior.